src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs

```diff
@@ -46,7 +46,7 @@ private void InitializeArgs(NativeCompileSettings config)
             argsList.Add($"\"{inputFilePath}\"");
             
             // System.Private.CoreLib Reference
-            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "System.Private.CoreLib.dll");
+            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "*.dll");
```

Comment:
Is this expected to change often? I am wondering if an enumeration would not be more reliable?